examples/blocks/readme.md

```diff
@@ -0,0 +1,31 @@
+# jsblocks • [TodoMVC](http://todomvc.com)
+
+From simple user interfaces to complex single-page applications using faster, server-side rendered and easy to learn framework
+
```

Comment:
Can we format this part of the readme as the other examples? (the quote on top and source + website)